www/src/components/ComponentApi.js

```diff
@@ -22,10 +22,13 @@ const Code = styled('code')`
 const propTypes = {};
 
 const Import = ({ name }) => (
```

Comment:
Move this into `CopyImport` to remove the duplication?

Reply:
Sure. I encapsulated both display and copy feature in that component and renamed it `ImportApi`. Is that okay?